versions/run.ver

```diff
@@ -1,4 +1,8 @@
 export evs_ver=v1.0
+# export met_ver=10.1.2
+# export metplus_ver=4.1.4
+export met_ver=11.0.0
```

Comment:
move the lines for met_ver=11.0.0 and metplus_ver=5.0.0 down to directly below netcdf_ver=4.7.4